pandas/tests/series/methods/test_explode.py

```diff
@@ -141,3 +143,25 @@ def test_explode_scalars_can_ignore_index():
     result = s.explode(ignore_index=True)
     expected = pd.Series([1, 2, 3])
     tm.assert_series_equal(result, expected)
+
+
+@pytest.mark.skipif(pa_version_under7p0, reason="minimum pyarrow not installed")
+def test_explode_pyarrow_list_type():
```

Comment:
Could you parameterize on `ignore_index` as well?

Reply:
updated